orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -443,6 +439,12 @@ def _convert_training_graph_input_to_list(self, *inputs, **kwargs):
                 inp = non_none_inputs[input_idx]
             elif name in kwargs and kwargs[name] is not None:
                 inp = kwargs[name]
+            elif input_idx >= len(non_none_inputs):
+                # Registered buffers are translated to user_input+initializer in ONNX
+                buffer_name, inp = next(self._original_module.named_buffers())
```

Comment:
IIUC this is always taking the *first* element from `self._original_module.named_buffers()`.

What happens if there are two or more registered buffers? Can you add unit tests for these cases?